### DISCUSSION TITLE
Contigs with length 128bp?? illumina read length is 250bp

### DISCUSSION THREAD

**Discussion by bala-ruokavirasto** [ablab/spades#790] [Q&A] [answered]
Hi Spades team,

I wonder why there is lesser length contig with 128bp in my spades assembly when the read length of my input illumina reads itself is 250bp? Can i improve the contig length by matching the contig to reads??  But why spades produce such a low length contig??

Best Regards,
Bala

**Comment by asl** [comment, marked as answer]
It's perfectly ok to have contigs of 128 bp. Usually they represent unresolved repeats.
